### PR TITLE
Adding taxonomical subheading

### PR DESCRIPTION
**Pull request by vprusso**
Closes: #658 

Adding a one-sentence description for each menu item.

### CODE REVIEW

**Review by WrathfulSpatula — APPROVED** [review]
If you make this something like a `<ViewSubHeader>` component instead of `<h5>`, just like `<ViewHeader>`, then we can change all instances in one place in code, if heading layout changes and `<h5>` is no longer appropriate. Otherwise, we'd have to string search for every instance of `<h5>`, if things change.

**Review by WrathfulSpatula — CHANGES_REQUESTED** [review]
See my earlier comment. (I hit the wrong option.)

**Comment by WrathfulSpatula** [comment]
Or, you could make it a string property of a `<ViewHeader>`. That way, it's inside of the containing `<header>` tag in that component.

**Comment by vprusso** [comment]
@WrathfulSpatula great suggestion. Makes sense. Done!

File: src/views/Methods.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ class Methods extends React.Component {
     return (
       <div id='metriq-main-content' className='container'>
         <ViewHeader>Methods</ViewHeader>
+          <ViewHeader>Methods are the algorithms used in a submission.</ViewHeader>
```
Comment (WrathfulSpatula) [inline src/views/Methods.js:82]: I think you meant to put this in `<ViewSubHeader>`.

File: src/views/Platforms.js [hunk 2]
```diff
@@ -79,6 +79,7 @@ class Platforms extends React.Component {
     return (
       <div id='metriq-main-content' className='container'>
         <ViewHeader>Platforms</ViewHeader>
+          <ViewHeader>Platforms are the hardware devices used for a submission.</ViewHeader>
```
Comment (WrathfulSpatula) [inline src/views/Platforms.js:82]: Same here: I think you meant to put this in `<ViewSubHeader>`.

**Pull request merged**